osf/models/mixins.py

```diff
@@ -1388,24 +1400,45 @@ def _get_admin_contributors_query(self, users, require_active=True):
             qs = qs.filter(user__is_active=True)
         return qs
 
-    def add_contributor(self, contributor, permissions=None, visible=True,
-                        send_email=None, auth=None, log=True, save=False, make_curator=False):
+    def add_contributor(
+            self,
+            contributor,
+            permissions=None,
+            visible=True,
+            notification_type=False,
+            auth=None,
+            log=True,
+            save=False,
+            make_curator=False
+    ):
         """Add a contributor to the project.
 
         :param User contributor: The contributor to be added
         :param list permissions: Permissions to grant to the contributor. Array of all permissions if node,
          highest permission to grant, if contributor, as a string.
         :param bool visible: Contributor is visible in project dashboard
-        :param str send_email: Email preference for notifying added contributor
+        :param str notification_type: Email preference for notifying added contributor
```

Comment:
Big change here, but it obviously a usability win to send notification type instead of inferred string, reduces complexity of notification type infference.